modules/storage-persistent-storage-efs-configmap.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// storage/persistent-storage/persistent-storage-efs.adoc
+
+[id="efs-creating-configmap_{context}"]
+= Creating a ConfigMap
+
+It is recommended to use a ConfigMap to contain all of the environment
+variables required for the EFS provisioner.
+
+.Procedure
+
+. Define an {product-title} ConfigMap containing the environment
+variables by creating a `configmap.yaml` with the following contents:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: efs-provisioner
+data:
+  file.system.id: <file-system-id> <1>
+  aws.region: <aws-region> <2>
+  provisioner.name: openshift.org/aws-efs <3>
+  dns.name: "" <4>
+----
+<1> Defines the Amazon EFS file system ID.
+<2> The Amazon region of the EFS file system, such as `us-east-1`.
+<3> The name of the provisioner for the associated StorageClass.
+<4> An optional argument that specifies the new DNS name where the EFS volume
+is located. If no DNS name is provided, the provisioner will search for the
+EFS volume at `<file-system-id>.efs.<aws-region>.amazonaws.com`.
+
+. Once the file has been configured, it may be created in your cluster
```

Comment:
s/Once/After
> ! once adv
Use only to mean “one time.” Do not use as a conjunction to mean “after” or “when.”

Do you need to do this to proceed? If so, s/it may be created/create it